terraform/install.sh

```diff
@@ -194,14 +194,16 @@ installMonitoring() {
     return 1
   fi
 
+  acct=$(gcloud info --format="value(config.account)")
```

Comment:
This line should be removed.  Acct is set in getProject().

Reply:
We need to call it again to retrieve the email for the Monitoring Terraform configuration